utils/files.ts

```diff
@@ -0,0 +1,46 @@
+import { RESPONSE_MESSAGES } from "../constants/index";
+import { NextApiResponse } from "next";
+import { Transform } from "stream";
+
+export function resolveHeaders(headers:string[]): string {
+    return headers.join(',');
+}
+
+export function getDataFromValues(data:any, headers: string[]): any {
```

Comment:
A better way to type this would be:
`export function getDataFromValues<T extends Record<string, any>>(data: T, headers: string[]): Partial<T> { ...`

then below you can define `result: Parital<T>` so we avoid using `any`.